### PR TITLE
fix(space): Add work item type group as a space relationship.

### PR DESCRIPTION
**Pull request by nimishamukherjee**
Introduce work item type group as a space relationship.

Dependent Planner PR https://github.com/fabric8-ui/fabric8-planner/pull/2380

https://docs.google.com/presentation/d/19vRm9GTG_3CEB89syBNzUrJvXZeak-KXs_WmpLd7CZE/edit

### CODE REVIEW

File: src/app/models/space.ts [hunk 1]
```diff
@@ -27,7 +27,8 @@ export class SpaceLink {
 export class SpaceRelationships {
     areas: SpaceRelatedLink;
     iterations: SpaceRelatedLink;
-    // this change breaks in fabric8-ui, fix it there to include this. 
+    workitemtypegroups: SpaceRelatedLink;
```
Comment (joshuawilson) [inline src/app/models/space.ts]: I think there are 2 tests that need to be updated too.

Comment (joshuawilson) [inline src/app/models/space.ts]: I think there are 2 tests that need to be updated too

File: src/app/spaces/space.service.spec.ts [hunk 2]
```diff
@@ -108,6 +108,11 @@ describe('Service: SpaceService', () => {
         iterations: {
           links: {
             related: 'http://example.com/api/spaces/1/iterations'
+          },
+          workitemtypegroups: {
```
Comment (joshuawilson) [inline src/app/spaces/space.service.spec.ts]: you made it a sub to iterations

Comment (nimishamukherjee) [reply src/app/spaces/space.service.spec.ts]: Ah sorry! Will fix it!

**Pull request merged**